File: POTD/2914.py
def solve(s):
  beauty=0
  for i in range(0,len(s)-1,2):
    if s[i]!=s[i+1]:
      beauty+=1
  return beauty
s=input()
if len(s)%2!=0:
  print("Invalid ODD String ")
else:
  print(solve(s))

File: POTD/2914.js
function beautycnt(string) {
  let beauty = 0;
  for (let i = 0; i < s.lenght - 1; i += 2) {
    if (s[i] != s[i + 1]) {
      beauty += 1;
    }
  }
  return beauty;
}
let string = prompt("Enter The String");
if (string.size % 2 != 0) console.log("Invalid ODD String");
else console.log(beautycnt(string));
